doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -5474,7 +5602,7 @@ equivalences are indeed taken into account, otherwise only single
 |SSR| searching tool
 --------------------
 
-.. cmd:: Search {? @pattern }  {* {? - } {| @string | @pattern } {? % @ident} } {? in {+ {? - } @qualid } }
+.. cmd:: Search {? {+ - {| @string {? % @ident } | @one_term } } } {? in {+ {? - } @qualid } }
```

Comment:
`Search` was removed from `ssr` and deprecated I think we should comment out this section.

Reply:
Why not just delete it?

Reply:
Ok